libs/databrickscfg/testdata/sample-home/.databrickscfg

```diff
@@ -0,0 +1,7 @@
+[DEFAULT]
+host = https://default
+token = default
+
+[acc]
+host = https://accounts.cloud.databricks.com
+account_id = abc
```

Comment:
Why do we need the new testdata file instead of using the existing one?

Reply:
because the parent folder uses a different name of config file (without a leading dot)